airflow-core/src/airflow/assets/manager.py

```diff
@@ -229,12 +242,39 @@ def notify_asset_alias_created(asset_assets: AssetAlias):
     def notify_asset_changed(asset: Asset):
         """Run applicable notification actions when an asset is changed."""
         try:
+            # todo: AIP-76 this will have to change. needs to know *what* happened to the asset (e.g. partition key)
```

Comment:
```suggestion
            # TODO: AIP-76 this will have to change. needs to know *what* happened to the asset (e.g. partition key)
```

nit: AFAIK, many editors highlight upper case `TODO`

Reply:
my editor (pycharm) doesn't care --- do some editors care?!?